bundles/org.openhab.binding.octopusenergy/src/main/java/org/openhab/binding/octopusenergy/internal/dto/Accounts.java

```diff
@@ -0,0 +1,102 @@
+/**
+ * Copyright (c) 2010-2021 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.octopusenergy.internal.dto;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.openhab.binding.octopusenergy.internal.exception.RecordNotFoundException;
+
+/**
+ * The {@link Accounts} class is a DTO to hold an API response to a 'accounts' call.
+ *
+ * @author Rene Scherer - Initial contribution
+ */
+@NonNullByDefault
+public class Accounts {
```

Comment:
This seems more than a DTO. You could move this class outside of the dto package (as checkstyle will treat it differently) or split this class in a DTO and a data conversion class.